example/android/app/src/main/kotlin/com/sap/gigya_flutter_plugin_example/MainActivity.kt

```diff
@@ -1,20 +1,5 @@
 package com.sap.gigya_flutter_plugin_example
 
-import com.gigya.android.sdk.account.models.GigyaAccount
-import com.sap.gigya_flutter_plugin.GigyaFlutterPlugin
 import io.flutter.embedding.android.FlutterActivity
-import io.flutter.embedding.engine.FlutterEngine
-import io.flutter.plugins.GeneratedPluginRegistrant
 
-class MainActivity : FlutterActivity() {
```

Comment:
This is an incorrect implementation.
You are removing the option to register a [custom scheme](https://sap.github.io/gigya-android-sdk/sdk-core/#providing-a-custom-account-schema
) account to the SDK, one of the core principles.

Reply:
Your proposed option uses an outdated approach and is not recommended by the Flutter team. I added another way.